encryptionkeyrotation/README.md

```diff
@@ -77,11 +77,16 @@ message EncryptionKeyRotateRequest {
   // This field is OPTIONAL
   string encryption_key = 3 [(csi.v1.csi_secret) = true];
 
+  // Volume capability describing how the CO intends to use this volume.
+  // This allows SP to determine if volume is being used as a block
+  // device or mounted file system. This is OPTIONAL.
+  csi.v1.VolumeCapability volume_capability = 4;
+
   // Plugin specific parameters passed in as opaque key-value pairs.
-  map<string, string> parameters = 4;
+  map<string, string> parameters = 5;
```

Comment:
you may not change the numbering of members in the message, you can only add new members with higher numbers

Reply:
I am aware of this, I did this as the implementation is not live yet. Shall I revert?